src/main/kotlin/com/faforever/userservice/domain/User.kt

```diff
@@ -61,3 +69,8 @@ interface UserRepository : ReactorCrudRepository<User, Int> {
     )
     fun findUserPermissions(userId: Int): Flux<Permission>
 }
+
+@R2dbcRepository(dialect = Dialect.MYSQL)
+interface AccountLinkRepository : ReactorCrudRepository<AccountLink, String> {
+    fun existsByUserIdAndOwnership(userId: Long, ownership: Boolean): Mono<Boolean>
```

Comment:
userId should be nullable

Reply:
Should this userId be nullable since I don't know if we want to allow an exists check by the null user since that is not really meaningful

Reply:
True, in this case it doesn't make sense.